cli/command/container/create.go

```diff
@@ -141,12 +142,7 @@ func pullImage(ctx context.Context, dockerCli command.Cli, image string, platfor
 	}
 	defer responseBody.Close()
 
-	return jsonmessage.DisplayJSONMessagesStream(
-		responseBody,
-		out,
-		dockerCli.Out().FD(),
-		dockerCli.Out().IsTerminal(),
-		nil)
+	return jsonmessage.DisplayJSONMessagesToStream(responseBody, streams.NewOut(out), nil)
```

Comment:
Hmm... actually this may not work, because `out` can be overridden to be `io.discard` if `--quiet` is set, and terminal detection would likely not be correct in that case, so detection must be done through the CLI's `Out()`, not `out` passed here.